src/plopp/plotting/common.py

```diff
@@ -86,6 +86,20 @@ def check_not_binned(obj):
         )
 
 
+def check_allowed_dtypes(obj):
+    forbidden = (
+        sc.DType.affine_transform3,
+        sc.DType.linear_transform3,
+        sc.DType.rotation3,
+        sc.DType.translation3,
+        sc.DType.vector3,
+    )
+    if obj.dtype in forbidden:
+        raise TypeError(
+            f'The input has dtype {obj.dtype} which is not supported by Plopp.'
+        )
```

Comment:
You could make this more proof for the future by using something like `obj.shape == obj.values.shape`, maybe?

Reply:
That is a nice suggestion :+1: